addons/website_quote/views/website_quotation.xml

```diff
@@ -507,6 +507,11 @@
                         </div>
                     </div>
                     <div class="col-md-9">
+                        <div class="row">
+                            <div groups="base.group_website_publisher" t-ignore="true" class="css_editable_mode_hidden col-md-offset-11">
```

Comment:
What is the link between being a webmaster and being able to come back to the quotation ?